runtime/compiler/control/J9Options.cpp

```diff
@@ -60,7 +60,22 @@
 #include "j9jitnls.h"
 #endif
 
+#if defined(__clang__)
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Winvalid-offsetof"
+#elif defined(__GNUC__) || defined(__GNUG__)
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Winvalid-offsetof"
+#endif
```

Comment:
It's not clear to me that the changes to this file will have any effect. The warnings need to be suppressed where `SET_OPTION_BIT()` is *used*, not where it is defined.

Reply:
The `-Winvalid-offsetof` warnings are seen in the last nightly build https://openj9-jenkins.osuosl.org/job/Build_JDK21_ppc64_aix_Nightly/347
but not the PR build https://openj9-jenkins.osuosl.org/job/Build_JDK21_ppc64_aix_Personal/255

Reply:
I see what Keith is talking about, because it doesn't look like this specific `SET_OPTION_BIT()` warning appears in the nightly build. I guess something must have happened since I was originally fixing this warning last year, and since I only checked to see if it got rid of the warnings, I missed that one of them doesn't do anything anymore.

Reply:
There are no warnings in the nightly that mention `SET_OPTION_BIT`, so I think that part of the change could be reverted.